### PR TITLE
Added Feats from Critical Role

### PR DESCRIPTION
**Pull request by Azzzzzy**

### CODE REVIEW

File: third-party/critical-role/taldorei-campaign-setting/feats.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                <li>Increase your Strength or Dexterity score by 1, to a maximum of 20.</li>
+                <li>Simple and martial melee weapons without the thrown property can be treated as if they have the thrown property. One-handed weapons have a range of 20/60, while two-handed weapons have a range of 15/30.</li>
+                <li>Weapons that naturally have the thrown property increase their range by +20/+40.</li>
+                <li>When you miss with a thrown weapon attack using a light weapon, the weapon immediately boomerangs back into your grasp.</li>
+            </ul>
+        </description>
+        <sheet>
+            <description>Simple and martial melee weapons can be thrown, one handed weapons with a range of 20/60 and two handed weapons with a range of 15/30. Thrown weapons gain additional range of +20/+40 and light thrown weapons that miss bomerang back to you.</description>
+        </sheet>
+        <rules>
```
Comment (swdriessen) [inline third-party/critical-role/taldorei-campaign-setting/feats.xml]: missing ability choice

Comment (Azzzzzy) [reply third-party/critical-role/taldorei-campaign-setting/feats.xml]: Wdym by ability choice? the feat gives either str or dex so wouldnt it be a choice?

Comment (swdriessen) [reply third-party/critical-role/taldorei-campaign-setting/feats.xml]: yeah, i am blind :bat:

**Comment by swdriessen** [comment]
Thanks

**Pull request merged**